### PR TITLE
Event calendar

### PR DESCRIPTION
**Pull request by fortress32501**
Adds the gem for the calendar
Implements the calendar in the events index page
- Supports adding events to the calendar
Adds an integration test for the calendar
Updates app to use CT instead of UTC

### CODE REVIEW

**Review by caleb-terry17 — APPROVED** [review]
looks great! does the following and nothing more!
- passed all integration and unit tests
- added gem and styling for calendar
- added calendar to events page for users and admin to view
- updated the time zone used by application to central from UTC
- added integration test to make sure calendar renders to page
- edited event unit tests to be in central time instead of UTC

File: app/views/events/index.html.erb [hunk 1]
```diff
@@ -37,3 +37,15 @@
...
+
+<!-- Render the calendar -->
+<%= month_calendar(events: @events, attribute: :event_start, end_attribute: :event_end) do |date, events| %>
+  <%= date.to_formatted_s(:short) %>
+
+  <!-- Render the events -->
+  <% events.each do |event| %>
+    <div style="border-style: solid;">
+      <%= link_to event.event_title, event %>
+    </div>
```
Comment (amaredia3) [inline app/views/events/index.html.erb:49]: When selecting events from calendar does it redirect to an events page where users can sign up?

**Comment by amaredia3** [comment]
Looks good! Checked for following:

- Functionality implemented according to the story assigned. 
- Code standard met:
   - Clear and concise variable naming
   - Proper code formatting formatting used (i.e whitespace, line length, file naming etc)

**Review by amaredia3 — APPROVED** [review]

File: app/views/events/index.html.erb (hunk 1 above)
Comment (fortress32501) [reply app/views/events/index.html.erb:49]: Not yet, the sign-up page is another user story so right now it redirects to the show page

**Pull request merged**